src/xunit.performance.run/ProgramCore.cs

```diff
@@ -137,11 +141,25 @@ private void RunTests(XunitPerformanceProject project)
             };
 
             startInfo.Environment["XUNIT_PERFORMANCE_RUN_ID"] = project.RunId;
-            startInfo.Environment["XUNIT_PERFORMANCE_MIN_ITERATION"] = "10";
-            startInfo.Environment["XUNIT_PERFORMANCE_MAX_ITERATION"] = "1000";
-            startInfo.Environment["XUNIT_PERFORMANCE_MAX_TOTAL_MILLISECONDS"] = "1000";
-            startInfo.Environment["COMPLUS_gcConcurrent"] = "0";
-            startInfo.Environment["COMPLUS_gcServer"] = "0";
+
+            if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("XUNIT_PERFORMANCE_MIN_ITERATION")))
+                startInfo.Environment["XUNIT_PERFORMANCE_MIN_ITERATION"] = MinIterations.ToString();
+
+            if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("XUNIT_PERFORMANCE_MAX_ITERATION")))
+                startInfo.Environment["XUNIT_PERFORMANCE_MAX_ITERATION"] = MaxIterations.ToString();
+
+            if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("XUNIT_PERFORMANCE_MAX_ITERATION_INNER_SPECIFIED")))
+                startInfo.Environment["XUNIT_PERFORMANCE_MAX_ITERATION_INNER_SPECIFIED"] = MaxIterationsWhenInnerIterationsSpecified.ToString();
+
+            if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("XUNIT_PERFORMANCE_MAX_TOTAL_MILLISECONDS")))
+                startInfo.Environment["XUNIT_PERFORMANCE_MAX_TOTAL_MILLISECONDS"] = MaxTotalMSPerTest.ToString();
+
+            // Disable server and background GC.
+            if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("COMPLUS_gcConcurrent")))
+                startInfo.Environment["COMPLUS_gcConcurrent"] = "0";
```

Comment:
I can guess Brian disabled the background GC because it has the potential increase noise, but I suspect the noise is not really an issue and is concerning because at least in some benchmarks you are trying to test the GC and don't want the harness forcing the config one way or the other.   

My same concern applies to the server GC. 

I would try to live without messing with the runtime modes.   Do we KNOW that they cause grief?